lib/packwerk/file_processor.rb

```diff
@@ -53,6 +53,9 @@ def call(relative_file)
       ProcessedFile.new(unresolved_references: unresolved_references)
     rescue Parsers::ParseError => e
       ProcessedFile.new(offenses: [e.result])
+    rescue StandardError => e
```

Comment:
What exactly is raising the error? The way this code is, if I write bad code in packwerk it is our users that will see the error, not us.